### ISSUE TITLE
Should "styles" property be "style" to match react-pdf components? 

### ISSUE THREAD

**Issue by jeffbski** [dmk99/react-pdf-table#9]
I got hung up on trying to use "style" property like we do with react-pdf components, but in react-pdf-table components you need to use "styles".

 Is this intentional to use `styles` rather than `style`? 

If possible it seems like it would be better to match react-pdf.

So currently this works

```jsx
<TableCell styles={styles.headerCell}>Week Start</TableCell>
```

but I'm thinking it would be better to match react-pdf so it's not as confusing when switching between them and not realizing why it isn't working. 

```jsx
<TableCell style={styles.headerCell}>Week Start</TableCell>
```

https://react-pdf.org/styling

**Comment by dmk99** [comment]
Yes, this makes a lot of sense. Sorry about the confusion around this, I will update the library to use 
`style`.

**Comment by jeffbski** [comment]
Awesome! Thanks.

**Comment by dmk99** [comment]
This should now be available in version `0.2.0`. I've also made the types align to what `react-pdf` expects e.g. a single `Style` or an array of `Style`s. 

Relevant documentation: https://github.com/dmk99/react-pdf-table/blob/master/docs/typedoc/interfaces/_tablecell_.tablecellprops.md#optional-styles
